webapp/loggedin/modules/users/userList/userListView.scss

```diff
@@ -1,5 +1,25 @@
-.user-list {
-  .table__row, .table__row-header {
+@import "../../../../style/vars";
+
+.users-list {
+  .table__rows {
+    cursor: pointer;
+  }
+
+  .table__row-header {
     grid-template-columns: repeat(4, 1fr);
```

Comment:
gridTemplateColumns is handled by TableView component, as a prop, passed by UsersListView; it's not necessary here.
Since the css hover style should be handled by the TableView component, this css is not needed.

Reply:
Right